src/components/ListSection.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { ListItem } from './ListItem';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTag } from '@fortawesome/fontawesome-free-solid';
+
+export const ListSection = ({ title, data, listToken, tagColor }) => {
+	return (
+		<>
+			<FontAwesomeIcon
+				icon={{
+					iconName: 'tag',
+					prefix: 'fas',
+				}}
+				style={{ color: tagColor, marginRight: 30 }}
+			/>
+			<h3>{title}</h3>
+			<ul>
+				{/* filter items be4 render */}
+				{data.map(({ name, ...items }) => (
+					<ListItem
+						key={items.id}
+						name={name}
+						items={items}
+						listToken={listToken}
+					/>
```

Comment:
I got confused trying to figure out what `items` means here. I thought it meant the list of items from the database, but it is something else. It's ok since it's been in use for a while in <ListItem />